src/python_inspector/resolution.py

```diff
@@ -15,6 +15,7 @@
 from typing import Sequence
 from zipfile import ZipFile
 
+import click
```

Comment:
There seems to be redundant changes in your three pending PR #42 , #43 and here?

Reply:
I have cut a branch from #43 here, will rebase once #43 gets merged